### PR TITLE
feat(web-iconics):about-us

### PR DESCRIPTION
**Pull request by BrookW-83**

### CODE REVIEW

File: Web/starter_project_iconics/components/about/A2svSessionCardItem.tsx [hunk 1]
```diff
@@ -0,0 +1,33 @@
+import React from "react";
+
+interface props {
```
Comment (zerihunMoges) [inline Web/starter_project_iconics/components/about/A2svSessionCardItem.tsx:3]: it is nice if you use PascalCase for interface and also a descriptive name

File: Web/starter_project_iconics/styles/globals.css [hunk 2]
```diff
@@ -15,3 +15,13 @@
     --background-end-rgb: 0, 0, 0;
   }
 }
+
+body {
```
Comment (Aklile-Yilma) [inline Web/starter_project_iconics/styles/globals.css:19]: why did you add styles to global css

**Review by zerihunmoges — DISMISSED** [review]

**Review by Aklile-Yilma — DISMISSED** [review]
LGTM
